resdk/tests/unit/test_resolwe.py

```diff
@@ -464,6 +464,32 @@ def test_good_response(self, resolwe_mock, requests_mock, os_mock, open_mock):
         # Why 6? 2 files in self.file_list, each downloads 3 chunks (defined in response mock)
 
 
+class TestDataUsage(unittest.TestCase):
+
+    @unittest.skipIf(six.PY2, "Skip in case of python2.x")
+    def test_data_usage(self):
```

Comment:
I doubt it is worth having such test as it is testing a really small part of whole functionality. It would be much better to write an e2e test that actually makes request to backend.